### PR TITLE
Fixes in eclipse-mosquitto image

### PR DESCRIPTION
**Pull request by ralight**
- Don't remove apk files
- Install mosquitto_passwd binary.

### CODE REVIEW

File: library/eclipse-mosquitto [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Maintainers: Roger Light <roger@atchoo.org> (@ralight)
 GitRepo: https://github.com/eclipse/mosquitto.git
-GitCommit: 1853bfc678c255367e7c2c2f138da7bf47054117
+GitCommit: 51738d41616aafc8a4b6632dfda8aaccc1d93d9f
```
Comment (tianon) [inline library/eclipse-mosquitto]: This like this was likely intended to be `da2879c33b6baf00b8b50a38da8d7f176cd85afa` instead? (`51738d41616aafc8a4b6632dfda8aaccc1d93d9f` exists, but doesn't appear to be part of any branches: https://github.com/eclipse/mosquitto/commit/51738d41616aafc8a4b6632dfda8aaccc1d93d9f)

Comment (ralight) [reply library/eclipse-mosquitto]: Yes, you're right. I remember fixing that but obviously didn't push it on to the PR. Sorry about the mixup.

**Comment by yosifkit** [comment]
<details>
<summary>Diff:</summary>

```diff
diff --git a/eclipse-mosquitto_latest/Dockerfile b/eclipse-mosquitto_latest/Dockerfile
index ef5418c..83c76d9 100644
--- a/eclipse-mosquitto_latest/Dockerfile
+++ b/eclipse-mosquitto_latest/Dockerfile
@@ -51,13 +51,14 @@ RUN set -x && \
     mkdir -p /mosquitto/config /mosquitto/data /mosquitto/log && \
 	install -d /usr/sbin/ && \
 	install -s -m755 /build/src/mosquitto /usr/sbin/mosquitto && \
+	install -s -m755 /build/src/mosquitto_passwd /usr/bin/mosquitto_passwd && \
 	install -m644 /build/mosquitto.conf /mosquitto/config/mosquitto.conf && \
     chown -R mosquitto:mosquitto /mosquitto && \
 	apk --no-cache add \
       libuuid \
       libwebsockets && \
 	apk del build-deps && \
-	rm -rf /build /etc/apk /lib/apk
+	rm -rf /build
 
 VOLUME ["/mosquitto/data", "/mosquitto/log"]
 
```

</details>

**Comment by yosifkit** [comment]
Build test of #5031; 77b49e4b; `amd64` (`eclipse-mosquitto`):

```console

$ bashbrew build eclipse-mosquitto:1.5.3
Building bashbrew/cache:605f94dd1ecbebd12e118952858f7191ef202be3fb72be46d7d5c78fbeb6c822 (eclipse-mosquitto:1.5.3)
Tagging eclipse-mosquitto:1.5.3
Tagging eclipse-mosquitto:1.5
Tagging eclipse-mosquitto:latest

$ test/run.sh eclipse-mosquitto:1.5.3
testing eclipse-mosquitto:1.5.3
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed

```

**Pull request merged**